resources/amperity/gocd/secret/vault/secrets-view.html

```diff
@@ -0,0 +1,26 @@
+<fieldset>
+    <legend style="font-size: 18px; margin-top: 15px;">Vault Plugin Configurations</legend>
+
+    <div class="form_item_block">
+        <label>Vault API<span class='asterix'>*</span></label>
+        <input type="text" ng-model="vault_addr" ng-required="true" placeholder="https://example.vault.url.amperity.com"/>
+        <span class="form_error" ng-show="GOINPUTNAME[vault_addr].$error.server">{{GOINPUTNAME[vault_addr].$error.server}}</span>
+    </div>
+
+    <div class="form_item_block">
+        <label>Authentication Method<span class='asterix'>*</span></label>
+        <form>
+            <select ng-model="auth_method" ng-required="true">
+                <option value="token">Token</option>
+            </select>
+        </form>
+        <span class="auth_method" ng-show="GOINPUTNAME[auth_method].$error.server">{{GOINPUTNAME[auth_method].$error.server}}</span>
+    </div>
+
+    <div class="form_item_mlock" ng-show="auth_method == 'token'">
+        <label>Vault Token, DO NOT USE IN PRODUCTION<span class='asterix'>*</span></label>
+        <input type="text" ng-model="vault_token" ng-required="auth_method == 'token'" placeholder=""/>
```

Comment:
Neat, so does this allow varying based on the selection from `auth_method`? I think that's true based on the PR description just wanted to check. Very nice if so, great job figuring that out. Its getting more hard over time trying to assert you are "not a ui person" ;) behold the ui wrangling!

Reply:
Yep!! Thanks :)